### ISSUE TITLE
No dark mode in topics?

### ISSUE THREAD

**Issue by KnowsCount** [isaacs/github#1915]
is it me or... Github Topics isn't in dark mode for me. For instance [Haskell](https://github.com/topics/haskell).

**Comment by filiptronicek** [comment]
Yep, also GitHub marketplace and the GitHub blog don't have dark mode.
